ui/src/app/views/workspace-list/component.ts

```diff
@@ -93,7 +93,7 @@ export class WorkspaceListComponent implements OnInit, OnDestroy {
   }
 
   addWorkspace(): void {
-    this.router.navigate(['workspace/build'], {relativeTo : this.route});
```

Comment:
Is it no longer relative route?

Reply:
nope relative in this context would make it workspaces/workspace/build. But workspace/build not a child of workspaces.